src/hypertrace/agent/instrumentation/aiohttp/__init__.py

```diff
@@ -116,8 +116,6 @@ async def on_request_end(
             params: aiohttp.TraceRequestEndParams,
     ) -> None:
         logger.debug('Entering hypertrace on_request_end().')
```

Comment:
Just curious, do we need this kind of logs?

Reply:
imo no